src/Dialog/Image/ImageDialog.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box } from '@mui/system';
+import Image from 'next/image';
+import React from 'react';
+
+export type ImageDialogProps = {
+  src: string;
+  alt: string;
+  fill?: boolean;
+  width?: number;
+  height?: number;
+}
+
+const ImageDialog: React.FC<ImageDialogProps> = ({ src, alt, fill, width, height }) => {
+  return (
+    <Box style={{ display: "flex", justifyContent: "center", position: 'relative' }}>
+      <Image
+        src={src}
+        alt={alt}
+        layout={fill ? "fill" : "intrinsic"}
+        objectFit="contain"
+        width={fill ? undefined : width}

```

Comment:
Remove this and line 22. Replace line 19 with `fill` (which is the same as `fill={true}`).